spec/services/badges/award_tag_spec.rb

```diff
@@ -61,4 +61,14 @@
     expect(second_user.reload.badge_achievements.size).to eq(1)
     expect(third_user.reload.badge_achievements.size).to eq(1)
   end
+
+  context "when award_tag_minimum_score setting is different than default" do
+    it "awards badge if qualifying article by score and tagged appropriately" do
+      allow(Settings::UserExperience).to receive(:award_tag_minimum_score).and_return(200)
```

Comment:
I would check the other situation - e.g. when `award_tag_minimum_score` is 50, and the article score is 51 => award a badge.
Currently it's 201 > 200, but if the code was not changed, and we would still be checking against 100, the badge still would be awarded 201 > 100.

Reply:
Cool, let me add this new test situation, thanks!